contracts/governance/GovernorAlpha.sol

```diff
@@ -361,11 +361,13 @@ contract GovernorAlpha is SafeMath96 {
 		uint96 totalFavorVotes = proposal.forVotes;
 		uint96 totalVotes = add96(totalFavorVotes, proposal.againstVotes, "GovernorAlpha:: state: forVotes + againstVotes > uint96");
 		uint96 favorVotesPercentage =
-			div96(
-				mul96(totalFavorVotes, 100, "GovernorAlpha:: state: mul error votes%"),
-				totalVotes,
-				"GovernorAlpha:: state: division error votes%"
-			);
+			totalFavorVotes > 0
+				? div96(
+					mul96(totalFavorVotes, 100, "GovernorAlpha:: state: mul error votes%"),
+					totalVotes,
+					"GovernorAlpha:: state: division error votes%"
+				)
+				: 0;
```

Comment:
looks like this zero division check was on purpose.
makes no sense to cancel if totalVotes == 0.
so it was reverting for correct reason and should anyway.

Reply:
Revert back